app/views/ingredients/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class='container wrapper'>
+  <div class='row'>
+    <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
```

Comment:
If the form below should be a part of this `div` then please nest it for better visibility.